index.js

```diff
@@ -728,6 +727,75 @@ module.exports = {
     })
   },
 
+  /**
+   * Gets a statistic on one field at a time
+   * Supports where and geometry filters and group by
+   * @param {String} table to get data from
+   * @param {String} field to generate stats from
+   * @param {String} outName the name of the stat field
+   * @param {String} type - the stat type: min, max, avg, count
+   */
+  getStat: function (table, field, outName, type, options, callback) {
+    // build sql
+    var fieldName
+    if (type === 'avg' || type === 'sum') {
+      fieldName = "(feature->'properties'->>'" + field + "')::int"
+    } else {
+      fieldName = "feature->'properties'->>'" + field + "'"
+    }
+    var fieldSql = type.toLowerCase() + '(' + fieldName + ')::int as "' + outName + '\"'
+
+    var groupByField = (options.groupby)
```

Comment:
what about multiple group by fields? seems like it's pretty easy to support that